mappings/map-af.tsv

```diff
@@ -180,12 +180,12 @@ ID	GLOSS	PRIORITY
 787	SAIL///seil	2
 1948	SAW///saag	4
 2783	SCALES (MEASURE INSTRUMENT)///skaal	2
-1328	SCISSORS///skêr	2
```

Comment:
Please check specifically this cases, where SCISSORS was still preferred in our mapping files, but should not be.